css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,24 @@
+button {
+  font-size: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-top: 22px;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+  padding-bottom: 22px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+  /* stylelint-disable-next-line function-url-quotes */
+  cursor: url(hand.cur), pointer;
```

Comment:
What does `url(hand.cur)` do? Please either explain how it works, or remove it. Also, please don't disable any `stylelint` rules unless an exercise specifically tells you to.

Reply:
Whenever the user hovers the button, the cursor changes to a pointer hand. I looked at the the instructions and checked out the link to cursor. I used what the examples provided.

The disable stylelint is there because for some reason im getting an error on my padding property like this: 
![image](https://user-images.githubusercontent.com/22688744/151232320-8751861c-812a-4bec-aa8b-075450d7c5e5.png)

Reply:
![css pseudo class 2](https://user-images.githubusercontent.com/22688744/151232859-0511d57e-30fa-4a35-815b-46b31e61a4f9.gif)

Final Output